### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Authentication_Keyboard: Users are unable to access "Anonymous", "Basic", "Digest" and "Windows" tabs using keyboard

### ISSUE THREAD

**Issue by InduPriya1805** [Microsoft/IIS.WebManager#93]
GitHubTags: #A11y_Manage.IIS.Net; #A11yMAS; #A11yTCS; #C+E-H; #DesktopWeb; #Win10; #.NETWebsites; #Edge; #Chrome; #Keyboard; #MAS2.1.1

**Environment Details:**
URL: https://manage.iis.net/webserver/authentication
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome 68.0.3440.106
Keyboard
MAS ID Violated: MAS 2.1.1, MAS 1.3.2

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till Web Server and press enter
5. Tab till Authentication and press enter
6. Verify whether "Anonymous", "Basic", "Digest" and "Windows" tabs are accessible using keyboard

**Actual Result:**
Users are unable to access "Anonymous", "Basic", "Digest" and "Windows" tabs using keyboard

**Expected Result:**
Users should able to access "Anonymous", "Basic", "Digest" and "Windows" tabs using keyboard

**Note:**
This issue has been observed in all the pages in the application which has tabs

**User Impact:**
Users with disabilities finds its difficult in working through the application if "Anonymous", "Basic", "Digest" and "Windows" tabs are not accessible using keyboard

**Attachments:**
[Bug93_web server_authentication_tabs in the page are not keyboard accessible.pptx](https://github.com/Microsoft/IIS.WebManager/files/2323382/Bug93_web.server_authentication_tabs.in.the.page.are.not.keyboard.accessible.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev1;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
